Patches/HLX Rimworld UNSC Armoury/Apparel_Headgear.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>[HLX] Rimworld: UNSC Armoury</li>
+    </mods>
+    <match Class="PatchOperationSequence">
+      <operations>
+
+        <!-- === Heavy Combat Helmet === -->
+        <li Class="PatchOperationReplace">
+          <xpath>/Defs/ThingDef[defName="RUA_Apparel_CombatHelmet"]/statBases/StuffEffectMultiplierArmor</xpath>
+          <value>
+            <Bulk>6</Bulk>
+            <WornBulk>4</WornBulk>
```

Comment:
This seems too high. Even the power armor helmets only have 1 wornBulk.

Reply:
Whoops my bad